vspace/src/main/java/edu/asu/diging/vspace/core/services/impl/SpaceManager.java

```diff
@@ -253,4 +254,14 @@ public void deleteSpaceById(String id) {
 
         return spaceLinkRepo.getLinkedFromSpaces(id);
     }
+    
+    @Override
+    public Iterable<Space> addIncomingLinkInfoToAllSpaces(Iterable<Space> allSpaces) {

```

Comment:
calling your list "allSpaces" is confusing, why would it have to be all?

Reply:
It is all spaces, and for each space that we display in the space list page ( which is all the spaces ), I add the boolean for if they have  incoming links or not.